src/main/java/step3/RacingCar.java

```diff
@@ -1,15 +1,18 @@
 package step3;
 
-import java.util.ArrayList;
 import java.util.List;
 import java.util.Random;
+import step3.domain.Car;
+import step3.domain.Position;
+import step3.domain.Winners;
+import step3.view.InputView;
+import step3.view.ResultView;
 
 public class RacingCar {
 
     private static final Random random = new Random();
     private static final InputView inputView = new InputView();
     private static final ResultView resultView = new ResultView();
```

Comment:
질문 주신 부분이네요!
컨벤션 확인에 관한 피드백의 의도는 static final 상수의 명명 규칙에 관한 내용이었습니다!
상수라는 것을 대문자와 언더바(_)를 통해서 구분해 주면 어떨까요?

```suggestion
    private static final InputView INPUT_VIEW = new InputView();
    private static final ResultView RESULT_VIEW = new ResultView();
```